backend/services/prisonerProfileService.js

```diff
@@ -5,14 +5,26 @@ const {
   apis: {
     categorisation: { ui_url: categorisationUrl },
     pathfinder: { ui_url: pathfinderUrl },
+    soc: { url: socUrl, enabled: socEnabled },
     useOfForce: { prisons: useOfForcePrisons, ui_url: useOfForceUrl },
   },
   app: { notmEndpointUrl, displayRetentionLink },
 } = require('../config')
 const logErrorAndContinue = require('../shared/logErrorAndContinue')
+const logger = require('../log.js')
 
-module.exports = ({ elite2Api, keyworkerApi, oauthApi, dataComplianceApi, pathfinderApi, systemOauthClient }) => {
+module.exports = ({
+  elite2Api,
+  keyworkerApi,
+  oauthApi,
+  dataComplianceApi,
+  pathfinderApi,
+  socApi,
+  systemOauthClient,
+}) => {
   const getPrisonerProfileData = async (context, offenderNo, username) => {
+    logger.info(`SOC API enabled: ${socEnabled}, SOC URl: ${socUrl}`)
```

Comment:
this will log the SOC information every time someone goes to prisoner profile, will will fill up the logs